invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -52,14 +52,15 @@ export class RecordCitationField extends Component {
   };
 
   fetchCitation = async (record, style) => {
-    return await axios(
-      `${record.links.self}?locale=${navigator.language}&style=${style}`,
-      {
-        headers: {
-          Accept: "text/x-bibliography",
-        },
-      }
-    );
+    const includeDeleted = record.deletion_status.is_deleted
+      ? "&include_deleted=1"
```

Comment:
I would try to avoid hardcoding it to be set to true. What if we don't pass it in the landing page view ? Why would it be set here? Maybe we can see if the original view has the argument in the URL somehow?

Reply:
It should not be possible to arrive at this cmp with a deleted record without passing `include_delete` param in the URL. However, I agree that it is better to get it from the view, I have adressed this. I am not sure how we can  avoid hardcoding the parameter in the URL